### PR TITLE
Security bugfix

### PR DESCRIPTION
**Pull request by Dev-Mba**
The fix of a crucial bug of infinite loop

### CODE REVIEW

**Comment by Vic-Soft** [comment]
Currently reviewing it now.

File: ahio.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+ayee
```
Comment (Vic-Soft) [inline ahio.txt:1]: Change "ayee" to "aye".
This must be done asap!

File: glitch.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+biggy boy
```
Comment (Vic-Soft) [inline glitch.txt:1]: Write it as "big boy".
This must also be done immediately.

File: papa.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+Papaya
```
Comment (Vic-Soft) [inline papa.txt:1]: This is good!

File: trd.txt [hunk 4]
```diff
@@ -0,0 +1 @@
+more
```
Comment (Vic-Soft) [inline trd.txt:1]: This is in tandem.

**Review by Vic-Soft — CHANGES_REQUESTED** [review]
Please attend to all changes in the review comment.

File: papa.txt (hunk 3 above)
Comment (Dev-Mba) [reply papa.txt:1]: Noted!

**Review by Vic-Soft — APPROVED** [review]

**Pull request merged**